Jenkinsfile

```diff
@@ -297,91 +300,102 @@ pipeline {
                 stage('E2E Upgrade') {
                     steps {
                         CreateCluster('upgrade')
-                        runTest('upgrade-haproxy', 'upgrade')
+                        runTest('upgrade-haproxy', 'upgrade', '8.0', 'upgrade-haproxy-8-0')
                         ShutdownCluster('upgrade')
                         CreateCluster('upgrade')
-                        runTest('upgrade-proxysql', 'upgrade')
+                        runTest('upgrade-proxysql', 'upgrade', '8.0', 'upgrade-proxysql-8-0')
                         ShutdownCluster('upgrade')
                         CreateCluster('upgrade')
-                        runTest('smart-update', 'upgrade')
-                        runTest('upgrade-consistency', 'upgrade')
+                        runTest('smart-update', 'upgrade', '8.0', 'smart-update-8-0')
+                        runTest('upgrade-consistency', 'upgrade', '8.0', 'upgrade-consistency-8-0')
                         ShutdownCluster('upgrade')
                     }
                 }
                 stage('E2E Basic Tests') {
                     steps {
                         CreateCluster('basic')
-                        runTest('haproxy', 'basic')
-                        runTest('init-deploy', 'basic')
-                        runTest('limits', 'basic')
-                        runTest('monitoring-2-0', 'basic')
-                        runTest('affinity', 'basic')
-                        runTest('one-pod', 'basic')
-                        runTest('auto-tuning', 'basic')
-                        runTest('proxysql-sidecar-res-limits', 'basic')
-                        runTest('users', 'basic')
-                        runTest('tls-issue-self','basic')
-                        runTest('tls-issue-cert-manager','basic')
-                        runTest('tls-issue-cert-manager-ref','basic')
-                        runTest('validation-hook','basic')
-                        runTest('proxy-protocol','basic')
+                        runTest('haproxy', 'basic', '8.0', 'haproxy-8-0')
```

Comment:
As for me it is really strange that we need to pass this  'haproxy-8-0' argument  because we already have 'haproxy' and '8.0'.

Reply:
yep, I know. As you can remember I tried many way and it is the only one which works) I don't like it too and agree to recheck it again.